gumby/experiments/isolated_community_loader.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from os import path
+from sys import path as pythonpath
+
+from gumby.experiments.community_launcher import CommunityLauncher
+from gumby.experiments.gumby_session import DefaultCommunityLoader
+
+# TODO(emilon): Fix this crap
+pythonpath.append(path.abspath(path.join(path.dirname(__file__), '..', '..', '..', "./tribler")))
+
+from Tribler.dispersy.crypto import ECCrypto
+
+
+class IsolatedLauncherWrapper(CommunityLauncher):
```

Comment:
Yes, I could combine it. However, this is a non-trivial extension of default CommunityLauncher behavior. Furthermore, this separates user launcher implementations from the test suite, which is desireable functionality in a test suite.